File: 1051-height-checker/1051-height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function(heights) {
    let result = 0;
    let orderedHeight = [...heights];
    orderedHeight.sort((a,b) => a - b);
    
    for(let index = 0; index < heights.length; index++) {
        if(heights[index] !== orderedHeight[index]) {
            result++;
        }
    }
    
    return result;
};

File: 1051-height-checker/1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] orderedHeight = heights.clone();        
        Arrays.sort(orderedHeight);
        int result = 0;
        
        for(int i = 0; i < heights.length; i++) {
            if(heights[i] != orderedHeight[i]) {
                result++;
            }
        }
        
        return result;
    }
}